<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="timeLineSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QToolButton" name="openAction">
            <property name="toolTip">
             <string>打开日志文件</string>
            </property>
            <property name="text">
             <string>打开</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/open.png</normaloff>:/res/img/open.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="gotoLineAction">
            <property name="toolTip">
             <string>定位到行</string>
            </property>
            <property name="text">
             <string>跳到行...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/locate.png</normaloff>:/res/img/locate.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="filterAction">
            <property name="toolTip">
             <string>过滤日志</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/filter.png</normaloff>:/res/img/filter.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="navBackAction">
            <property name="toolTip">
             <string>后退</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/left.png</normaloff>:/res/img/left.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="navAheadAction">
            <property name="toolTip">
             <string>前进</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/right.png</normaloff>:/res/img/right.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SearchEdit" name="searchEdit">
            <property name="placeholderText">
             <string>输入文本进行搜索</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="caseSensitivityCheckBox">
            <property name="text">
             <string>大小写敏感</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="logSplitter">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>2</number>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="logContainer">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="LogTextEdit" name="logEdit">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollBar" name="logEditVBar">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="subLogContainer">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="LogTextEdit" name="subLogEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollBar" name="subLogEditVBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="clipboardAction">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/clipboard.png</normaloff>:/res/img/clipboard.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="exportTimeLineAction">
            <property name="toolTip">
             <string>导出时间线到图片</string>
            </property>
            <property name="text">
             <string>导出时间线...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/img/export.png</normaloff>:/res/img/export.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="TimeLine" name="timeLine" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menuEncoding">
     <property name="title">
      <string>编码(&amp;E)</string>
     </property>
     <addaction name="actionGBK"/>
     <addaction name="actionUTF8"/>
    </widget>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
    <addaction name="menuEncoding"/>
    <addaction name="separator"/>
    <addaction name="actionsetting"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menuInsight">
    <property name="title">
     <string>检视(&amp;I)</string>
    </property>
    <addaction name="actionsearch"/>
    <addaction name="actionreverseSearch"/>
    <addaction name="actionsearchBack"/>
    <addaction name="actionsearchFoward"/>
    <addaction name="separator"/>
    <addaction name="actionfilter"/>
    <addaction name="separator"/>
    <addaction name="actiongoBack"/>
    <addaction name="actiongoFoward"/>
    <addaction name="actiongotoLine"/>
   </widget>
   <widget class="QMenu" name="menuTimeline">
    <property name="title">
     <string>时间线(&amp;T)</string>
    </property>
    <addaction name="actiontoClipboard"/>
    <addaction name="actionexport"/>
    <addaction name="separator"/>
    <addaction name="actionclearAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionshortcut"/>
    <addaction name="actiondoc"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInsight"/>
   <addaction name="menuTimeline"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>打开...(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="text">
    <string>查找...(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionsearchBack">
   <property name="text">
    <string>查找上一个(&amp;U)</string>
   </property>
  </action>
  <action name="actionsearchFoward">
   <property name="text">
    <string>查找下一个(&amp;D)</string>
   </property>
  </action>
  <action name="actionfilter">
   <property name="text">
    <string>过滤...(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actiongoBack">
   <property name="text">
    <string>后退(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="actiongoFoward">
   <property name="text">
    <string>前进(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>设置...(&amp;S)</string>
   </property>
  </action>
  <action name="actionclearAll">
   <property name="text">
    <string>清空(&amp;D)</string>
   </property>
  </action>
  <action name="actionexport">
   <property name="text">
    <string>保存为图片...(&amp;S)</string>
   </property>
  </action>
  <action name="actiontoClipboard">
   <property name="text">
    <string>复制到剪贴板(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionshortcut">
   <property name="text">
    <string>查看快捷键...(&amp;S)</string>
   </property>
  </action>
  <action name="actiondoc">
   <property name="text">
    <string>帮助文档...(&amp;T)</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于...(&amp;A)</string>
   </property>
  </action>
  <action name="actiongotoLine">
   <property name="text">
    <string>跳到行...(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGBK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GBK</string>
   </property>
  </action>
  <action name="actionUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionreverseSearch">
   <property name="text">
    <string>反向查找(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogTextEdit</class>
   <extends>QTextEdit</extends>
   <header>logtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>TimeLine</class>
   <extends>QWidget</extends>
   <header>timeline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QLineEdit</extends>
   <header>searchedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
